content/actions/guides/building-and-testing-powershell.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Building and testing PowerShell
+intro: You can create a continuous integration (CI) workflow to build and test your PowerShell project.
+versions:
+  free-pro-team: '*'
+  enterprise-server: '>=2.22'
+---
+
+{% data reusables.actions.enterprise-beta %}
+{% data reusables.actions.enterprise-github-hosted-runners %}
+
+### Introduction
+
+This guide shows you how to install, test and publish a PowerShell module.
```

Comment:
> install, test and publish a PowerShell module.

Does install here refer to installing dependencies?

Reply:
great point! yes. perhaps it should be how to `install dependencies, test your module, and publish to the PowerShell Gallery`?

Reply:
That sounds good to me 👍

Reply:
Done, @rachmari ! 🚀